ReactiveCocoa/UIKit/iOS/UIRefreshControl.swift

```diff
@@ -0,0 +1,31 @@
+import ReactiveSwift
+import enum Result.NoError
+import UIKit
+
+extension Reactive where Base: UIRefreshControl {
+	/// Sets whether the refresh control should be refreshing.
+	public var isRefreshing: BindingTarget<Bool> {
+		return makeBindingTarget { $1 ? $0.beginRefreshing() : $0.endRefreshing() }
+	}
+
+	/// Sets the attributed title of the refresh control.
+	public var attributedTitle: BindingTarget<NSAttributedString?> {
+		return makeBindingTarget { $0.attributedTitle = $1 }
+	}
+
+	/// The action to be triggered when the refresh control is refreshed. It
+	/// also controls the enabled state of the refresh control.
+	public var refreshed: CocoaAction<Base>? {
+		get {
+			return associatedAction.withValue { info in
+				return info.flatMap { info in
+					return info.controlEvents == .valueChanged ? info.action : nil
+				}
+			}
+		}
+
+		nonmutating set {
+			setAction(newValue, for: .valueChanged)
```

Comment:
@mdiep Is there a way to implement that without copying all of `Reactive<UIControl>.setAction(for:)`? None of the other `Reactive<UIControl>` extensions attempt to add extra bindings to their actions.